docs/src/api/commandline.md

```diff
@@ -0,0 +1,221 @@
+`uniffi-bindgen-react-native` the command line utility that ties together much of the building of Rust, and the generating the bindings and turbo-modules.
+
+Most commands take a `--config FILE` option. This is a YAML file which collects commonly used options together, and is [documented here](config-yaml.md).
+
+The command can be invoked with `./node_modules/.bin/uniffi-bindgen-react-native`.
+
+Running the following command from within your project:
+
+```sh
+alias ubrn=$(pwd)/node_modules/.bin/uniffi-bindgen-react-native
+```
+allows you to run the command as `ubrn`, which is simpler to type. From hereon, commands will be given as `ubrn` commands.
+
+# `ubrn` Commands
+
+Running `ubrn --help` gives the following output:
+
+```sh
+Usage: uniffi-bindgen-react-native <COMMAND>
+
+Commands:
+  checkout  Checkout a given Github repo into `rust_modules`
+  build     Build (and optionally generate code) for Android or iOS
+  generate  Generate bindings or the turbo-module glue code from the Rust
+  help      Print this message or the help of the given subcommand(s)
+
+Options:
+  -h, --help  Print help
+```
+
+## `checkout`
+Checkout a given Git repo into `rust_modules`.
+
+```sh
+Usage: uniffi-bindgen-react-native checkout [OPTIONS] <REPO>
+
+Arguments:
+  <REPO>  The repository where to get the crate
+
+Options:
+      --config <CONFIG>
+      --branch <BRANCH>  The branch or tag which to checkout [default: main]
+  -h, --help             Print help
+```
+The checkout command can be operated in two ways, either:
+1. with a `REPO` argument and optional `--branch` argument. OR
+2. with a config file which may specify a repo and branch, or just a `directory`.
```

Comment:
It's implied, but might be worth mentioning that you can just include a rust crate under rust_modules/{NAME}/Cargo.toml instead of executing this step.